cql3/statements/select_statement.hh

```diff
@@ -243,7 +243,8 @@ private:
             service::query_state& state,
             const query_options& options,
             gc_clock::time_point now,
-            lw_shared_ptr<const service::pager::paging_state> paging_state) const;
+            lw_shared_ptr<const service::pager::paging_state> paging_state,
+            uint32_t internal_page_size) const;
 
```

Comment:
Let's remove DEFAULT_COUNT_PAGE_SIZE from .hh, since it's no longer used.

Reply:
Yes, sorry, I missed it.